### ISSUE TITLE
Episode 14 tidyr: gap_wide_betterID

### ISSUE THREAD

**Issue by sysilviakim** [swcarpentry/r-novice-gapminder#687]
Good afternoon! In line 36 of the `14-tidyr.Rmd`, it says
```
write.csv(gap_wide_betterID,"data/gapminder_wide.csv",row.names = FALSE)
```
but considering lines 28 to 33, it should be simply
```
write.csv(gap_wide, "data/gapminder_wide.csv", row.names = FALSE)
```
I think it would benefit from applying the styler package to the `gap_wide` code in general, as it seems inconsistent in spacing. Leaving here as an issue rather than a PR for future checkouts.

**Comment by jcoliver** [comment]
Indeed. The comment on line 36 of the RMarkdown file should indicate the object being saved is `gap_wide`, not `gap_wide_betterID`.

**Comment by nung001** [comment]
Hi @sysilviakim thanks for the opportunity and great catch. If it's alright I'd be happy to take this one. Going to make the change in the comments and see where the code could benefit from the some 'styler' aesthetic magic, i do see some inconsistencies that @sysilviakim had mentioned. I'll send a PR when done. @jcoliver do the carpentries have a preferred r style guide or is the default 'tidyverse_style' alright? I'm doing this for my checkout and this is my first time contributing so I'll do my best to follow protocols here, thanks for bearing with me.

**Comment by jcoliver** [comment]
Addressed by #695 .
